assets/scss/modules/_box.scss

```diff
@@ -1,4 +1,10 @@
-@mixin dp-box ($color, $width, $max-width, $min-height){
+
+/// Box component
+/// We can use the following parameters to customize them $color, $width, $max-width, $min-height
+/// @access private
+/// @author gbaamonde@makingsense.com
+
+@mixin dp-box ($color:"", $width:"", $max-width:"", $min-height:"") {
```

Comment:
What do you think about using common default values?

Reply:
You're right, now I add the default values